common/src/main/resources/wynntils.mixins.json

```diff
@@ -2,22 +2,23 @@
   "client": [
     "AbstractContainerScreenMixin",
```

Comment:
I thought these are sorted by spotless... At least, they used to be

Reply:
They don't seem to be properly sorted right now anyway, and spotless has not complained. However, I see that the IntelliJ sort I used seem to be case-sensitive and that the old sort was not:

```
-    "ClientPacketListenerMixin",
     "ClientPackSourceMixin",
+    "ClientPacketListenerMixin",
```

I don't know which is "right", or at least "best". I think it is convenient to use "sort lines" in IntelliJ, but non-case sensitive sorting seems saner, I think.